script.js

```diff
@@ -0,0 +1,136 @@
+// Start here
+
+// Step 1 - Welcome and introduction
+// Your code goes here
+alert(
+  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
```

Comment:
It would be fun if you personalise this first alert, just to make it interesting, make up the name for a pizza restaurant?